### PR TITLE
chore: migrate PasswordInput

### PR DESCRIPTION
**Pull request by anucreative**
## COMPATIBILITY

- [ ] Tested on Safari (desktop)
- [ ] Tested on Chrome (desktop)
- [ ] Tested on Firefox (desktop)
- [ ] Tested on mobile device sizes
- [ ] Tested on tablet device sizes
- [ ] Tested on IOS Safari (either device or simulator)

## QA

- [ ] Thoroughly tested in local environment
- [ ] Added tests for all new features
- [ ] Added tests that considered edge cases

### CODE REVIEW

File: lib/src/components/PasswordInput/ToggleButton.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+export type ToggleButtonProps = React.ComponentPropsWithoutRef<typeof Button> & {
+  isHidden: boolean
+}
+
+export const ToggleButton = ({ isHidden, onClick, title, ...rest }: ToggleButtonProps) => {
+  return (
+    <Button
+      aria-controls="password"
+      aria-expanded={`${!isHidden}`}
```
Comment (cnairi) [inline lib/src/components/PasswordInput/ToggleButton.tsx]: I think `aria-expanded={isHidden}` is the same here?

**Review by cnairi — APPROVED** [review]
